torch/nn/modules/module.py

```diff
@@ -1181,7 +1181,10 @@ def load_state_dict(self, state_dict: 'OrderedDict[str, Tensor]',
 
         Args:
             state_dict (dict): a dict containing parameters and
-                persistent buffers.
+                persistent buffers. To load a dict from a DataParallel or
+                DistributedDataParallel module, need to first apply
+                :meth:`~torch.nn.modules.utils._consume_prefix_in_state_dict_if_present`
```

Comment:
Is this intentional to expose a private function in a stable API doc?

Reply:
Indeed, we either want to make this a public function or not document it. But not both.

Reply:
Made it public. Thanks!